contracts/multisig-prover/src/msg.rs

```diff
@@ -8,19 +8,38 @@ use crate::encoding::{Data, Encoder};
 
 #[cw_serde]
 pub struct InstantiateMsg {
+    // Address that can call UpdateWorkerSet
```

Comment:
/// Address that can execute all messages that either have unrestricted or admin permission level, such as UpdateWorkerSet. Should be set to a trusted address that can react to unexpected interruptions to the contract's operation.